server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -1398,7 +1398,7 @@ exports.updateOperationKey = function (body, user, originator, xCorrelator, trac
             isUpdated = await operationServerInterface.setOperationKeyAsync(operationUuid, newOperationKey);
           }
         } else {
-          throw new Error("OperationServerUuidisnotPresent")
+          reject(new BadRequestHttpException("OperationServerUuidisnotPresent"))
```

Comment:
Shall we make the reson for rejection 
from 
OperationServerUuidisnotPresent
to 
OperationServerUuid is not present

Reply:
done